scripts/install-dev.sh

```diff
@@ -348,30 +333,53 @@ install_deno_vscode_extension() {
         fi
 }
 
+terms_of_service() {
+        echo -e ""
+        echo -e "📄 Use of the Slack CLI should comply with the Slack API Terms of Service:"
+        echo -e "🏛️  https://slack.com/terms-of-service/api"
+}
+
 feedback_message() {
         CODE=$?
-        if [ $CODE -eq 0 ] && [ $(command -v $SLACK_CLI_NAME) ]; then
-                echo -e "\n💌 We would love to know how things are going. Really. All of it."
-                echo -e "   Survey your development experience with \`$SLACK_CLI_NAME feedback\`"
+        echo -e "\x1b[0m"
+        echo -e "💌 We would love to know how things are going. Really. All of it."
+        if command -v "$SLACK_CLI_NAME" >/dev/null 2>&1; then
+                echo -e "✨ Survey your development experience with \`$SLACK_CLI_NAME feedback\`"
         else
-                echo -e "\x1b[0m"
-                echo -e "💌 We would love to know how things are going. Really. All of it."
-                echo -e "   Submit installation issues: https://github.com/slackapi/slack-cli/issues"
-                exit $CODE
+                echo -e "✨ Submit installation issues: https://github.com/slackapi/slack-cli/issues"
         fi
-}
-
-terms_of_service() {
-        if [ $(command -v $SLACK_CLI_NAME) ]; then
-                echo -e "\n📄 Use of the Slack CLI should comply with the Slack API Terms of Service:"
-                echo -e "   https://slack.com/terms-of-service/api"
+        if [ $CODE -ne 0 ]; then
+                exit $CODE
         fi
 }
 
 next_step_message() {
-        if [ $(command -v deno) ] && [ $(command -v $SLACK_CLI_NAME) ]; then
-                echo -e "\n✨ You're all set! Next, authorize your CLI in your workspace with \`$SLACK_CLI_NAME login\`\n"
+        echo -e ""
+        if command -v "$SLACK_CLI_NAME" >/dev/null 2>&1; then
+                echo -e "📺 Success! The Slack CLI (build: $SLACK_CLI_DEV_VERSION) is now installed!"
+        else
+                echo -e "📝 To get started, manually add the Slack CLI to your shell path:"
+                case "$(basename "$SHELL")" in
+                bash)
+                        echo -e "   echo 'export PATH=\"\$HOME/.local/bin:\$PATH\"' >> ~/.bashrc"
+                        echo -e "   source ~/.bashrc"
+                        ;;
+                fish)
+                        echo -e "   mkdir -p \$HOME/.config/fish"
+                        echo -e "   echo 'fish_add_path \$HOME/.local/bin' >> \$HOME/.config/fish/config.fish"
+                        echo -e "   source \$HOME/.config/fish/config.fish"
+                        ;;
+                zsh)
+                        echo -e "   echo 'export PATH=\"\$HOME/.local/bin:\$PATH\"' >> ~/.zshrc"
+                        echo -e "   source ~/.zshrc"
+                        ;;
+                *)
+                        echo "   export PATH=\"$local_bin_path:\$PATH\""
+                        ;;
+                esac
```

Comment:
🔍 note: Here's a combined output of successful install from above, and next steps per shell!